wg-node-lifecycle/charter.md

```diff
@@ -0,0 +1,109 @@
+# WG Node Lifecycle Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+The Kubernetes ecosystem currently faces challenges in node maintenance scenarios, with multiple
+projects independently addressing similar issues. The goal of this working group is to develop
+unified APIs that the entire ecosystem can depend on, reducing the maintenance burden across
+projects and addressing scenarios that impede node drain or cause improper pod termination. Our
```

Comment:
Is it the plan to add a unified API which control the kubernetes features in this area ? ( like Priority classes, PDBs..etc? ) 
If Pod scaling ..etc are in consideration , do we also need to consider Resource Quota and Auto scalers like HPA here?